app/routes/sites/companies/company/edit.js

```diff
@@ -1,15 +1,22 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  model() {
+    return Ember.RSVP.hash({
+      company: this.modelFor('sites.companies.company'),
+      sites: this.modelFor('sites.companies.company').get('networkSites')
+    });
+  },
+
   actions: {
     updateCompany(name) {
-      const company = this.modelFor(this.routeName);
+      const company = this.modelFor('sites.companies.company');
```

Comment:
can you pass the model as an argument so you don't have to rely on the other route for this model?